THCalendarDatePicker/THDatePickerViewController.m

```diff
@@ -674,7 +674,10 @@ - (BOOL)dateInFutureAndShouldBeDisabled:(NSDate *)dateToCompare {
     currentDate = [calendar dateFromComponents:[calendar components:comps fromDate:currentDate]];
     dateToCompare = [calendar dateFromComponents:[calendar components:comps fromDate:dateToCompare]];
     NSComparisonResult compResult = [currentDate compare:dateToCompare];
-    return (compResult == NSOrderedDescending && _daysInHistory && _daysInHistory <= dayDifference) || (compResult == NSOrderedAscending && _daysInFuture && _daysInFuture <= dayDifference);
```

Comment:
This I don't get... actually its working how I had it... In this constellation it is not giving me correct results (I,ve set daysinfuture to 0 (=disabled) and could not set any days